src/App.js

```diff
@@ -106,9 +133,10 @@ function App() {
             const data = await res.json()
             // Create cookie
             setCookie(
-                    'user',
-                    { 'username': signUpInfo.username, 'token': data.token },
-                    { path: '/' }
+                'user',
+                { 'username': signUpInfo.username, 'token': data.token },
```

Comment:
If we are going to allow users to be logged in after signing up, also pass in the admin field, since the admin field is used to hide certain pages. You can reference loginUser().